stdlib/builtins.pyi

```diff
@@ -1041,16 +1041,16 @@ class filter(Iterator[_T], Generic[_T]):
 
 def format(__value: object, __format_spec: str = ...) -> str: ...  # TODO unicode
 @overload
-def getattr(__o: object, name: str) -> Any: ...
+def getattr(__o: object, __name: str) -> Any: ...
 
 # While technically covered by the last overload, spelling out the types for None and bool
 # help mypy out in some tricky situations involving type context (aka bidirectional inference)
 @overload
-def getattr(__o: object, name: str, __default: None) -> Any | None: ...
+def getattr(__o: object, __name: str, __default: None) -> Any | None: ...
```

Comment:
Interesting that type checkers didn't complain about a positional-only arg after a positional-or-keyword one.

Reply:
Yeah I also thought that was strange!